setup.py

```diff
@@ -8,6 +8,8 @@
     "brainio_collection @ git+https://github.com/brain-score/brainio_collection",
     # test_requirements
     "pytest",
+    "pandas",
+    "imageio",
```

Comment:
to avoid these piling up, can you please put the dependencies specific to your assemblies/stimulus_sets in `mkgu_packaging/dicarlo/marques/setup.py`?

Reply:
What information should I put in the setup file? Should I describe this particular package as brainio_contrib.dicarlo.marques ?

Reply:
it can be very minimal, a `requirements.txt` with one line per dependency would also be sufficient. If a setup.py, I don't think you have to put any info in it except the requirements. It's just for people to reproduce/retrace your packaging if need be

Reply:
Done